### ISSUE TITLE
ChartJs

### ISSUE THREAD

**Issue by CamelJohn** [laravel-frontend-presets/black-dashboard#1]
I'm trying to figure out how to use your preset (witch is awesome) to understand how i can create more views of my own - styled the same way - while creating new charts as well as passing data to those charts VIA controller functions that i can write on my own. lets say using variables that conatin query data from different models i use in my app. 

as soon as i try to change the HTML canvas or the corespondet JS filel all the charts go blank.

also, is there a place where i can learn about the slugging method you use, because the routing system you use is more advance than my method and I'm afraid as soon as i create a controller it will be difficult to render a certain view with charts that work.

**in short** how can i create charts using your code, in another view, and pass data to them from a controller i will create

thank you for your help

**Comment by teamupdivision** [comment]
Hello @CamelJohn , thanks for using our products!
The charts are done with https://www.chartjs.org/.
You can check their docs here https://www.chartjs.org/docs/latest/

Thanks
